tests/headers/use-core.h

```diff
@@ -1,4 +1,4 @@
-// bindgen-flags: --use-core --raw-line "extern crate core;" --with-derive-hash --with-derive-partialeq --with-derive-eq
+// bindgen-flags: --use-core --raw-line '#![cfg(not(target_os="windows"))] extern crate core;' --with-derive-hash --with-derive-partialeq --with-derive-eq
```

Comment:
Why doesn't this build?

Reply:
```
     Running tests\use-core.rs (target\debug\deps\use_core-a332e4538c4c2cf1.exe)

running 2 tests
test bindgen_test_layout__bindgen_ty_1 ... FAILED
test bindgen_test_layout_foo ... ok

failures:

---- bindgen_test_layout__bindgen_ty_1 stdout ----
thread 'bindgen_test_layout__bindgen_ty_1' panicked at 'assertion failed: `(left == right)`
  left: `4`,
 right: `8`: Size of: _bindgen_ty_1', tests\use-core.rs:62:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

which is 

```rust
#[repr(C)]
#[derive(Copy, Clone)]
pub union _bindgen_ty_1 {
    pub bar: ::std::os::raw::c_int,
    pub baz: ::std::os::raw::c_long,
}
assert_eq!(
        ::core::mem::size_of::<_bindgen_ty_1>(),
        8usize,
        concat!("Size of: ", stringify!(_bindgen_ty_1))
    );
```

which when looking into it is really strange. This fails on the last assert

```rust
#[repr(C)]
#[derive(Copy, Clone)]
pub union _bindgen_ty_1 {
    pub bar: ::std::os::raw::c_int,
    pub baz: ::std::os::raw::c_long,
}

#[test]
fn test() {
    assert_eq!(::std::mem::size_of::<::std::os::raw::c_long>(), 4);
    assert_eq!(::std::mem::size_of::<::std::os::raw::c_int>(), 4);
    assert_eq!(4 + 4, 8);
    assert_eq!(::std::mem::size_of::<_bindgen_ty_1>(), 8);
}
```